doc/classes/Basis.xml

```diff
@@ -214,7 +214,8 @@
 			<description>
 				Creates a new [Basis] with a rotation such that the forward axis (-Z) points towards the [param target] position.
 				By default, the -Z axis (camera forward) is treated as forward (implies +X is right). If [param use_model_front] is [code]true[/code], the +Z axis (asset front) is treated as forward (implies +X is left) and points toward the [param target] position.
-				The up axis (+Y) points as close to the [param up] vector as possible while staying perpendicular to the forward axis. The returned basis is orthonormalized (see [method orthonormalized]). The [param target] and [param up] vectors cannot be [constant Vector3.ZERO], and cannot be parallel to each other.
+				The up axis (+Y) points as close to the [param up] vector as possible while staying perpendicular to the forward axis. The returned basis is orthonormalized (see [method orthonormalized]).
+				It is recommended that the direction from the node position to the [param target] vector not be parallel to the [param up] vector. Although, the Z axis points to the target, but the second axis is not defined, therefore causing a Z rotation implicitly.
```

Comment:
```suggestion
				It is recommended that the direction from the node position to the [param target] vector not be parallel to the [param up] vector. In this case the Z axis points to the target the second axis is not defined, therefore causing a Z rotation implicitly.
```
Assuming this was meant, also should clarify if it's `+Z` or `-Z` that points to the target

Reply:
It can be either, depending on the use_model_front argument